core/src/main/java/org/kiwix/kiwixmobile/core/help/HelpScreen.kt

```diff
@@ -0,0 +1,205 @@
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+
+import androidx.navigation.NavController
+import org.kiwix.kiwixmobile.core.R
+import org.kiwix.kiwixmobile.core.error.DiagnosticReportActivity
+import org.kiwix.kiwixmobile.core.extensions.ActivityExtensions.start
+
+val SendDiagnosticReportFontSize = 18.sp
+
+@Composable
+fun HelpScreen(
+  modifier: Modifier = Modifier,
+  data: List<HelpScreenItemDataClass>,
+  navController: NavController
+) {
+  val isDarkTheme = isSystemInDarkTheme()
+  val backgroundColor =
+    if (isDarkTheme) colorResource(id = R.color.mine_shaft_gray900) else Color.White
+  val dividerColor =
+    if (isDarkTheme) colorResource(id = R.color.mine_shaft_gray600)
+    else colorResource(id = R.color.mine_shaft_gray350)
+
+  Scaffold(
+    modifier = Modifier.fillMaxSize(),
+    topBar = {
+      Row(
+        modifier = Modifier.fillMaxWidth(),
+        verticalAlignment = Alignment.CenterVertically
+      ) {
+        HelpTopAppBar(navController)
+      }
+    },
+    containerColor = backgroundColor
+  ) { innerPadding ->
+    HelpContent(data, dividerColor, innerPadding)
+  }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun HelpTopAppBar(navController: NavController) {
+  // Retrieve the actionBarSize from the current theme
+  val context = LocalContext.current
+  val actionBarHeight = with(LocalDensity.current) {
+    // Obtain the height defined in the theme (usually 56dp on phones)
+    val styledAttributes =
+      context.theme.obtainStyledAttributes(intArrayOf(android.R.attr.actionBarSize))
+    styledAttributes.getDimension(0, 0f).toDp().also { styledAttributes.recycle() }
+  }
+
+  TopAppBar(
+    modifier = Modifier.height(actionBarHeight), // set the height here
+    title = {
+      Row(
+        modifier = Modifier.fillMaxSize(),
+        verticalAlignment = Alignment.CenterVertically
+      ) {
+        Text(
+          modifier = Modifier.padding(
+            start = dimensionResource(R.dimen.activity_horizontal_margin)
+          ),
+          text = stringResource(id = R.string.menu_help),
+          color = Color.White,
+          fontWeight = FontWeight.SemiBold
+        )
+      }
+    },
+    navigationIcon = {
+      Row(modifier = Modifier.fillMaxHeight(), verticalAlignment = Alignment.CenterVertically) {
+        IconButton(onClick = navController::popBackStack) {
+          Icon(
+            imageVector = Icons.AutoMirrored.Filled.ArrowBack,
+            contentDescription = "Back_Navigation",
+            tint = Color.White
+          )
+        }
+      }
+
+    },
+    colors = TopAppBarDefaults.topAppBarColors(
+      containerColor = Color.Black
+    )
+  )
+}
+
+@Composable
+fun HelpContent(
+  data: List<HelpScreenItemDataClass>,
+  dividerColor: Color,
+  innerPadding: androidx.compose.foundation.layout.PaddingValues
```

Comment:
Instead of writing full `packageName` just import this package and use the `PaddingValues` directly.